src/utils/hooks/api/loans/use-get-account-positions.tsx

```diff
@@ -59,31 +68,74 @@ const getAccountPositionsStats = (
 
   const borrowAmountUSD = getPositionsSumOfFieldsInUSD('amount', borrowPositions, prices);
 
-  const collateralLendPositions = lendPositions
-    .filter(({ isCollateral }) => isCollateral)
-    .map(({ amount, currency, ...rest }) => ({
-      // MEMO: compute total value based on collateral threshold (not full lend amount value)
-      amount: amount.mul(assets[currency.ticker].collateralThreshold),
-      currency,
-      ...rest
-    }));
+  const collateralLendPositions = lendPositions.filter(({ isCollateral }) => isCollateral);
+
+  const collaterAmountPositions = collateralLendPositions.map(({ amount, currency, ...rest }) => ({
+    // MEMO: compute total value based on collateral threshold (not full lend amount value)
+    amount: amount.mul(assets[currency.ticker].collateralThreshold),
+    currency,
+    ...rest
+  }));
 
-  const collateralAmountUSD = getPositionsSumOfFieldsInUSD('amount', collateralLendPositions, prices);
+  const liquidationLendPositions = collateralLendPositions.map(({ amount, currency, ...rest }) => ({
```

Comment:
Need to improve name, what about `collateralPositionsWithAppliedCollateralThreshold` - quite long, but explicit? 
Same could be done for L80: `collateralPositionsWithAppliedLiquidationThreshold`

Reply:
If you have better naming idea, I would love it :)